charts/spire/charts/spire-agent/values.yaml

```diff
@@ -281,3 +281,12 @@ experimental:
   syncInterval: 5s
   ## @param experimental.featureFlags [array] List of developer feature flags
   featureFlags: []
+
+sockets:
+  ## @param sockets.hostBasePath Path on which the agent socket is made available when admin.expose is true
+  hostBasePath: /run/spire/agent/sockets
```

Comment:
I don't think there is a valid use case for putting the admin socket on the host.

Reply:
Addressed above